docs/reference/aggregations/bucket/rare-terms-aggregation.asciidoc

```diff
@@ -174,7 +174,23 @@ which match the `max_doc_count` criteria will be returned.  The aggregation func
 the order-by-ascending issues that afflict the `terms` aggregation.
 
 This does, however, mean that  a large number of results can be returned if chosen incorrectly.
-To limit the danger of this setting, the maximum `max_doc_count` is 10.
+To limit the danger of this setting, the maximum `max_doc_count` is 100.
+
+[[search-aggregations-bucket-rare-terms-aggregation-max-buckets]]
+==== `search.max_buckets`
```

Comment:
Nit: I'm not a fan of heading that are settings names, could we maybe call this something like "Max bucket limit"?